app/stac_api/urls.py

```diff
@@ -40,4 +43,7 @@
         AssetDetail.as_view(),
         name='asset-detail'
     ),
+    # https://www.django-rest-framework.org/tutorial/quickstart/#urls
+    path(f"{API_BASE}/api-auth/", include('rest_framework.urls', namespace='rest_framework')),
+    path('api-token-auth/', obtain_auth_token, name='api-token-auth'),
```

Comment:
Perhaps it might make sense to have the rest_framework.urls directly at `API_BASE` and drop the `api-auth` part. The login URL would then be `API_BASE/login/`.
And perhaps the obtain_auth_token endpoint could also be under `API_BASE/api-token-auth/` or `API_BASE/obtain-token/`.
What do you think?

Reply:
For the `api-toke-auth` I think as well that `API_BASE/get_token/` or sth makes sense. I'm wondering whether the rest of the `api-auth` part is even necessary, since Django itself provides the login facilities?

Reply:
I would keep all endpoints at the same base, this is for the user easier to navigate. Moreover if we need in the future to give the login to external users, we could add a link to it in the landing page and we would have a consistent URL path. I would also move the admin page in the `API_BASE`.

Reply:
The reason for the admin page to be not in API_BASE was, that it should be independent of the current version. Hence `/v0.9/` was dropped. I think, that still makes sense, doesn't it?
For the rest I'll make it consistent and check, if the `api-auth` part is necessary.

Reply:
> For the `api-toke-auth` I think as well that `API_BASE/get_token/` or sth makes sense. I'm wondering whether the rest of the `api-auth` part is even necessary, since Django itself provides the login facilities?

Yes, the `include('restframework.urls', ...)` is necessary. Otherwise there'll be no `log-in` resp. `log-out` link on the nav-bar in the browsable API.